catalog/imaging/imaging-actionprovider-chip/pom.xml

```diff
@@ -76,11 +74,41 @@
                 </exclusion>
             </exclusions>
         </dependency>
+        <dependency>
+            <groupId>org.webjars.bower</groupId>
+            <artifactId>components-bootstrap</artifactId>
+            <version>3.1.1</version>
+        </dependency>
+        <dependency>
+            <groupId>org.webjars.bower</groupId>
+            <artifactId>components-font-awesome</artifactId>
+            <version>4.6.3</version>
+        </dependency>
+        <dependency>
+            <groupId>org.webjars.bower</groupId>
+            <artifactId>jquery</artifactId>
+            <version>1.11.0</version>
```

Comment:
I put those in the root pom to resolve transitive dependency version conflicts. I think the move to webjars is a stopgap solution until we can move to yarn+browserify/webpack. Keeping the versions in the pom makes it easier to migrate to yarn so we don't have to compute an effective pom to resolve versions.